src/dev/pci/PciUpstream.py

```diff
@@ -51,11 +51,13 @@ class PciBus(NoncoherentXBar):
 
     config_error_port = RequestPort("Port to send config errors to")
 
-    # Set some default values bases on IOXBar
+    # FIXME: There should have some latency added to transfert packets within
```

Comment:
Rather than putting this in the definition of the *model*, let's move it to the stdlib where we define the parameters to the models. (As a side note, I really don't like that we set so many default parameters like this where we define the models. I would much rather have a nice separation of concerns between the definitions of how the models function and the parameters that we use.)

So, if we could move this to the X86Board, that would be a better place, IMO. Especially since it seems that only X86 is affected.